curator/README.md

```diff
@@ -0,0 +1,24 @@
+# Curator AI - Newsletter Mail Agent
+
+Mailing part of the Newsletter. Uses the curate agent to generate the content and formats the data to send it in an email. 
+
+## Send a sample email
+
+First, define the POSTMARK_SERVER_API_KEY and POSTMARK_DEFAULT_MAIL in .env located in curator workspace root :
+
+```txt
+POSTMARK_SERVER_API_KEY=XXX
+POSTMARK_DEFAULT_MAIL=XXX
```

Comment:
**suggestion:** Since this requires a anv file, can you add an `.env.sample` file to your repository with all environment variables to set for the project to run ?

Bonus: the `.env` file can be created via a `cp -n .env.sample .env` when running `make send_mail`